### PR TITLE
Davidcit

### PR DESCRIPTION
**Pull request by davidcit646**
**Added**
Socom variant MATVs
unarmed MATVs for platoon/godfather elements

**Removed**
NOTHING!

**Fixed**
Pilot spawning without uniform (#6)
Team leaders & squad leaders spawning with M4s without acog RMRs
manually applied ace medical levels to mustang elements (#3)

**Updated**
cScripts medical settings
cba_settings

### CODE REVIEW

**Comment by davidcit646** [comment]
Awaiting review

File: cba_settings.sqf [hunk 1]
```diff
@@ -118,20 +139,22 @@ force force ace_repair_displayTextOnRepair = true;
...
+force force ace_magazinerepack_timePerBeltLink = 1;
 force force ace_magazinerepack_timePerMagazine = 2;
 
 // ACE Map
-force force ace_map_BFT_Enabled = true;
-force force ace_map_BFT_HideAiGroups = true;
-force force ace_map_BFT_Interval = 5;
+force force ace_map_BFT_Enabled = false;
+force force ace_map_BFT_HideAiGroups = false;
+force force ace_map_BFT_Interval = 1;
```
Comment (BojanOnTheInternet) [inline cba_settings.sqf:157]: Do we really want to full disable the BFT?

File: cba_settings.sqf [hunk 2]
```diff
@@ -156,55 +179,52 @@ force force ace_maptools_rotateModifierKey = 1;
...
-force force ace_medical_useLocation_basicEpi = 0;
-force force ace_medical_useLocation_PAK = 0;
-force force ace_medical_useLocation_SurgicalKit = 0;
-
-// ACE Mk6 Mortar
-force force ace_mk6mortar_airResistanceEnabled = false;
-force force ace_mk6mortar_allowCompass = true;
-force force ace_mk6mortar_allowComputerRangefinder = false;
-force force ace_mk6mortar_useAmmoHandling = false;
+force force ace_medical_spontaneousWakeUpChance = 0.05;
```
Comment (BojanOnTheInternet) [inline cba_settings.sqf:198]: Suggest this is increased to 0.15, which gives you a 47% of getting up after 1 minutes, 85% of getting up after 3 minutes, 96% of getting up after 5 minutes. Keep in mind people still need to get to a "stable" state first, which may take some time as well

File: cba_settings.sqf [hunk 3]
```diff
@@ -249,16 +269,16 @@ force force ace_finger_maxRange = 10;
 // ACE Pylons
 force force ace_pylons_enabledForZeus = true;
 force force ace_pylons_enabledFromAmmoTrucks = true;
-force force ace_pylons_rearmNewPylons = true;
+force force ace_pylons_rearmNewPylons = false;
```
Comment (BojanOnTheInternet) [inline cba_settings.sqf:272]: This should be `true` so its not cancer changing pylons

File: cba_settings.sqf [hunk 4]
```diff
@@ -249,16 +269,16 @@ force force ace_finger_maxRange = 10;
...
 force force ace_pylons_requireToolkit = true;
 force force ace_pylons_searchDistance = 15;
-force force ace_pylons_timePerPylon = 10;
+force force ace_pylons_timePerPylon = 5;
 
 // ACE Quick Mount
 force force ace_quickmount_distance = 5;
-force force ace_quickmount_enabled = false;
-ace_quickmount_enableMenu = 3;
+force force ace_quickmount_enabled = true;
```
Comment (BojanOnTheInternet) [inline cba_settings.sqf:280]: This should be `false` to avoid conflicts with vehicle permissions, once that's brought in

File: cba_settings.sqf [hunk 5]
```diff
@@ -410,222 +440,108 @@ force force acex_viewrestriction_modeSelectiveSea = 0;
...
-// Achilles - Modules Interface
-//Achilles_var_moduleTreeCollapse = true;
-//Achilles_var_moduleTreeDLC = true;
-//Achilles_var_moduleTreeHelmet = false;
+acex_volume_enabled = false;
+acex_volume_fadeDelay = 1;
+acex_volume_lowerInVehicles = false;
+acex_volume_reduction = 5;
+acex_volume_remindIfLowered = false;
+acex_volume_showNotification = true;
```
Comment (BojanOnTheInternet) [inline cba_settings.sqf:448]: Suggest we leave this out, let players use their own settings

File: cba_settings.sqf [hunk 6]
```diff
@@ -410,222 +440,108 @@ force force acex_viewrestriction_modeSelectiveSea = 0;
...
+//acre_sys_list_SwitchChannelColor = [0.66,0.05,1,1];
+//acre_sys_list_ToggleHeadsetColor = [0.66,0.05,1,1];
 
 // ACRE2 Zeus
-acre_sys_zeus_zeusCanSpectate = true;
-acre_sys_zeus_zeusDefaultVoiceSource = false;
-acre_sys_zeus_zeusListenViaCamera = false;
+force force acre_sys_zeus_zeusCanSpectate = true;
+force force acre_sys_zeus_zeusCommunicateViaCamera = true;
+force force acre_sys_zeus_zeusDefaultVoiceSource = false;
```
Comment (BojanOnTheInternet) [inline cba_settings.sqf:483]: Allow clients to set this, remove from here

**Review by BojanOnTheInternet — CHANGES_REQUESTED** [review]

**Comment by Jaidendude** [comment]
Updated parent pull request to reflect issue tracker numbers: #6 #3

**Pull request merged**